Update quickbooks.md (#538)
cyclr/docs: pages/v2/05-application-connectors/authentication/quickbooks.md

{::options parse_block_html="true" /}
<section class="card">

## QuickBooks setup 

> > This only needs to be completed one time.  Once your details are entered into the console you will be able to use them for all of your customers, without any need for them to be involved.

Reviewer: ```suggestion
> This setup only needs to be completed once. You will be able to use the details entered into the console for all your customers, without any need for them to be involved, once they are entered. 
```
Slight word reorder.
